tests/new_upgrades/test_subscription.py

```diff
@@ -173,40 +168,11 @@ def subscription_auto_attach_setup(
         sat_upgrade.ready()
         yield test_data
 
-
-@pytest.mark.subscription_upgrades
-@pytest.mark.rhel_ver_list([7, 8, 9, 10])
-@pytest.mark.no_containers
-@pytest.mark.manifester
-def test_subscription_scenario_auto_attach(subscription_auto_attach_setup):
```

Comment:
As you're removing this test for newer upgrades scenarios, could we also drop it for older upgrade scenarios? Sorry to expand the scope of this PR a bit, just thought it might be a good opportunity for some additional cleanup,

https://github.com/SatelliteQE/robottelo/blob/master/tests/upgrades/test_subscription.py#L77